src/error.cpp

```diff
@@ -160,7 +160,7 @@ namespace {
           N_("invalid type in tiff structure") },
         { Exiv2::kerInvalidMalloc,
           N_("invalid memory allocation request") },
-        { Exiv2::kerCorruptedMetadata,
+        { Exiv2::kerTiffParsingError,
           N_("corrupted image metadata") },
```

Comment:
I think we should adjust the error message, too. How about something highly creative like: "Tiff parsing error: corrupted image metadata"?

Reply:
No need to change the error message since I reverted this rename.